logistics_project/apps/api/resources/v0_1.py

```diff
@@ -218,18 +218,19 @@ class Meta(CustomResourceMeta):
 
 
 class StockTransactionResources(ModelResource):
-    supply_point = fields.ToOneField(SupplyPointResources, 'supply_point', full=True, null=True)
+    supply_point = fields.IntegerField('supply_point_id', null=True)
 
     def dehydrate(self, bundle):
-        if bundle.obj.product_report:
-            bundle.data['report_type'] = bundle.obj.product_report.report_type
-        else:
-            bundle.data['report_type'] = None
-
         bundle.data['product'] = bundle.obj.product.sms_code
-        bundle.data['supply_point'] = bundle.obj.supply_point.id
         return bundle
 
+    def apply_filters(self, request, applicable_filters):
```

Comment:
In django 1.3 it's not possible to filtering by supply_point_id so that's why I added this method. I'm not sure what version is used on prod.